clap_builder/src/error/format.rs

```diff
@@ -458,6 +409,35 @@ fn write_dynamic_context(
     }
 }
 
+#[cfg(feature = "error-context")]
+fn write_possible_values(
+    styled: &mut StyledStr,
+    valid: &anstyle::Style,
+    possible_values: Option<&ContextValue>,
+    list_name: &'static str,
```

Comment:
nit: if this is used for things besides possible values, it should have a more general name